content/posts/introduce-dishwasher-part-1.md

```
@@ -0,0 +1,60 @@
+
+ということで置くタイプの食洗機から選択しよう、ということになった。東京ガスの方にはお帰りいただいた。ありがとう。
+
+## 置くタイプの食洗機を入れるにも色々ある
+
+置くタイプの食洗機をチョイスしていく。置くタイプの食洗機にも色々タイプがある。水をどこから調達するか、というので「タンク式」と「水栓から取る式」が存在する。タンクを使うやつは前もって水を補充しておいた状態で食洗機を動かす。水栓から水を取るやつは、使うときに自動的に水を調達してくれる。後者のほうが世話なしである。当然後者のものを選ぶことをファーストチョイスとしていく。
+
+## 分岐水栓がつけられない蛇口
+
+さて、水栓から取るってどういうことか。これは典型的には「水道の蛇口の根本のあたりにアタッチメントをつけて、水を分岐できるようにする」というやり方で実現するようだ。これを実現するためのアイテムが「分岐水栓」と呼ばれるブツである。これを購入して蛇口にくっつけることで (ちょっと工事じみたことをする)、水道を二股に分けられるようにするという寸法。
+
+ただ、分岐水栓というのも色々あって、我が家で使っている蛇口に適合するものを調達しないといけない。蛇口の型番を入れると適合する分岐水栓を探してくれるサイト (https://sumai.panasonic.jp/dishwasher/bunki/search.html) なんかもあったりする。
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)